src/main/java/com/corndel/nozama/repositories/UserRepository.java

```diff
@@ -32,6 +32,28 @@ public static List<User> findAll() throws SQLException {
 
   public static User findById(int id) throws SQLException {
     // TODO: finish this method
-    return null;
+    var query = "SELECT id, username, firstName, lastName, email, avatar FROM users WHERE id = ?";
+
+    try (var con = DB.getConnection();
+         var stmt = con.prepareStatement(query)) {
+      stmt.setInt(1, id);
+
+      try (var rs = stmt.executeQuery()){
+
+        if (!rs.next()) {
+          return null;
+      }
+
+        id = rs.getInt("id");
```

Comment:
I think this is just replacing id with the same thing, you might not need this line